Source/Particles/Gather/GetExternalFields.H

```diff
@@ -12,7 +12,9 @@
 #include <AMReX_Parser.H>
 #include <AMReX_REAL.H>
 
-enum ExternalFieldInitType { Constant, Parser };
+using namespace amrex::literals;
```

Comment:
Oh, we should not do `using namespace` in global scope of (header) files.
Can you move this into a body (i.e., into the `{...}` that use this)?

I think this would work if moved into `GetExternalField::operator ()`

Reply:
Ok, I'll move it.